ODF-community.french.md

```diff
@@ -0,0 +1,526 @@
+ • Publier les risques et les limitations découvertes en chemin <br/><br/>
+
+<strong>Imaginer l'échec</strong><br/>
+ • Imaginez que c'est le jour de l'annonce et que des personnes sont surprises
+ou ennervées. Qu'est-ce qui a déclenché cela ?<br/><br/>
+ • Identifier les  changement que vous feriez ou les points que vous pourriez
+clarifier et faites-les de manière pro-active.<br/><br/>
+
+<strong>Activer vos ambassadeurs</strong><Br/>
+ • Equiper la communauté pour qu'elle vous aide à clarifier les désinformations
+et les incompréhensions. 
+</td>
+<td>
+ • Est-il possible de faire un prototype ou publier très tôt
+afin de récolter des réactions ?<br/><br/>
+ • Comment allons-nous tester ?<br/><Br/>
+ • Quel usagers internes peuvent nous aider à tester ?<br/><br/>
+ • Est-ce qu'un groupe de travail transversal a du sens ?<br/><br/>
+ • Peut-on batir une communauté de passionnés autour de ce projet ou cette décision ?<br/><br/>
+ • Avons-nous impliqué les personnes qui vont devoir faire le travail ?<br/><br/>
+ • Auprès de qui devons nous chercher plus d'adhésion ou de soutien. 
+</td>
+</tr>
+<tr>
+<td><strong>Points clés</strong></td>
+</tr>
+<tr>
+<td>
+ • Representation de  différent types d'usagers<br/>
+ • Impacts et cas d'utilisation imprévus<br/>
+ • Risques et doutes non exprimés 
+</td>
+</tr>
+</table>
+
+<br/>
+
+Phase or activity: Launcer, Deployer, Cloturer <a name="launch"></a>
+--------------
+
+_page 10 sur 15_ 
+
+<table>
+<tr>
+<td colspan=2 width="66%"><strong>Étapes pour devenir ouvert</strong></td>
+<td><strong>Questions à poser</strong></td>
+</tr>
+<tr valign="top">
+<td rowspan=3 width=33%><strong>Commencer par envisager la fin</strong><br/>
+ • Demontrer l'alignment avec la stratégie, la mission, la culture et les
+veleurs de Red Hat<br/><br/>
+ • Souligner les étapes que vous avez suivie pour prendre votre décision de
+manière ouverte<br/><br/>
+ • Mettre en avant l'utilisation de ce cadre de décision ouverte <br/><br/>
+ • Dire aux collaborateurs ou trouver des informations détaillées<br/><br/>
+ • Monter comment le feedback a modelé la décision ou le projet<br/><br/>
+ • Expliquer comment fournir du feedback après le lancement<br/><br/>
+ • Signaler quand vous n'êtes pas complètement satisfait de la décision ou que
+vous savez que d'autres ne le seront pas. <br/><br/>
+ • Parteger votre feuille de route et vos critères pour revenir sur la décision<br/><br/>
+ • Rester en contact avec les personnes qui rejettent la décision
+</td>
+<td rowspan=3 width=33%>
+<strong>Ouvert par défaut</strong><br/>
+ • Réaffirmer les contraintes et les pré-requis pertinents <br/><br/>
+ • Partager les problèmes légaux, les rapports, et les
+questions de confidentialité concernés<br/><br/>
+ • Communiquer les critères de  succès et publier les métriques pertinentes<br/><br/>
+
+<strong>Contribuer en amont</strong><br/>
+ • Publier vos méthodes, les leçons apprises, les communiqués et les critères
```

Comment:
I propose some reformulations:
- `les méthodes` instead of `vos méthodes`
- `Publier les archives des méthodes` or `Archiver et publier`